http/internal/logic/social/followLogic.go

```diff
@@ -24,7 +32,78 @@ func NewFollowLogic(ctx context.Context, svcCtx *svc.ServiceContext) *FollowLogi
 }
 
 func (l *FollowLogic) Follow(req *types.FollowRequest) (resp *types.FollowResponse, err error) {
-	// todo: add your logic here and delete this line
-
-	return
+	UserClaims, err := utils.ParseToken(req.Token, l.svcCtx.Config.JwtAuth.Secret)
+	if err != nil {
+		return nil, schema.ApiError{
+			StatusCode: 422,
+			Code:       42204,
+			Message:    "token解析失败",
+		}
+	}
+	_, err = l.svcCtx.UserRpc.QueryById(l.ctx, &user.QueryByIdRequest{
+		UserId: req.ToUserId,
+	})
+	if _, match := utils.MatchError(err, common.ErrUserNotFound); match {
```

Comment:
here you'd better handle some unexpected errors, cause there may be database failure